src/workerd/io/compatibility-date.capnp

```diff
@@ -308,13 +308,12 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   brotliContentEncoding @32 :Bool
       $compatEnableFlag("brotli_content_encoding")
+      $compatEnableDate("2024-04-29")
```

Comment:
Does it matter this will not go out until next week?

Reply:
Since this just adds a compatibility date and not a flag, it should be fine as long as the documentation is updated in advance – concisely documenting what the flag does is difficult, but I'm finishing touches on it now.

Reply:
I've been reading to many release numbers I read the date as tomorrow.